tools/cldr-code/src/main/java/org/unicode/cldr/util/BNF.java

```diff
@@ -45,17 +45,17 @@ public String getInternal() {
     + "sequence = (core quantifier*)+;"
     + "alternation = sequence (weight? ('|' sequence weight?)+)?;"
     + "rule = string '=' alternation;";
-
-
+    
+    
     *      Match 0 or more times
```

Comment:
Something's broken here. Lines shouldn't end with tabs or spaces.

Reply:
Hmm good point.

These are lines within comments though, and arguably the comments should be properly blocked.
I suspect the Google formatter doesn't care about this case because unstructured comments aren't really allowed internally.